demos/yolopv2/README.md

```diff
@@ -0,0 +1,21 @@
+# YOLOPv2 Example
+
+Simplest possible example of tracking generic objects. Based on [YOLOPv2](https://github.com/CAIC-AD/YOLOPv2).
```

Comment:
Is it the simplest possible example?

Are we really tracking generic objects? In what dataset was YOLOpv2 trained on?

Reply:
We should mention this model does Panoptic Driving Perception.